css/index.css

```diff
@@ -47,3 +47,49 @@ div{
     width: 100%;
     height: 100%;
 }
+
+/* Format search results. 
+ */
+ul#search-results a { 
```

Comment:
since ids are unique, there's no reason to prepend ul
I prefer camelCase class names and IDs